engine/src/testing-libgdf.cu

```diff
@@ -219,8 +219,8 @@ static result_pair systemCommand(uint64_t accessToken, Buffer&& requestPayloadBu
 
       std::thread([]() {
           std::this_thread::sleep_for(std::chrono::seconds(1));
-          ral::communication::network::Client::closeConnections();
-          ral::communication::network::Server::getInstance().close();      
+          ral::communication::network::experimental::Client::closeConnections();
```

Comment:
This entire file is no longer used it can be deleted right?

Reply:
good catch ... deleted!